src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.c

```diff
@@ -21,45 +39,343 @@ const ActorInit Bg_Spot15_Rrbox_InitVars = {
     (ActorFunc)BgSpot15Rrbox_Update,
     (ActorFunc)BgSpot15Rrbox_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot15_Rrbox/func_808B3960.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot15_Rrbox/func_808B39E8.s")
+InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
```

Comment:
from jenkins logs (you can see the build failed)
```
mips-linux-gnu-ld: build/src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.o: in function `func_808B3960':
/tmp/preprocessedshotctf5.c:69: multiple definition of `sInitChain'; build/src/overlays/actors/ovl_Bg_Haka_Water/z_bg_haka_water.o:/tmp/preprocessedhqpcr98w.c:45: first defined here
make: *** [Makefile:158: zelda_ocarina_mq_dbg.elf] Error 1
```
the issue is due to both this and `z_bg_haka_water` using the same name `sInitChain`, you can solve it on your side by adding the `static` keyword (reference other actors like z_arrow_fire if you need an example) which makes the symbol (name) only defined in the current file.

Reply:
I've applied that change.